cerberus_anymal_b_control_1/CMakeLists.txt

```diff
@@ -0,0 +1,102 @@
+cmake_minimum_required(VERSION 3.10)
+project(cerberus_anymal_b_control_1)
+
+## Use C++14
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+## Package dependencies
+set(PACKAGE_DEPENDENCIES
+  catkin_simple
+  cerberus_anymal_control_base
+  cerberus_anymal_control_ros
+  cerberus_anymal_utils
+  geometry_msgs
+  glog_catkin
```

Comment:
I didn't find any place where we're using glog. Might be wrong, though. Can be removed?